analytics/terraform/spark-k8s-operator/addons.tf

```diff
@@ -3,7 +3,7 @@
 #---------------------------------------------------------------
 module "eks_data_addons" {
   source  = "aws-ia/eks-data-addons/aws"
-  version = "~> 1.30" # ensure to update this to the latest/desired version
+  version = "~> 1.34" # ensure to update this to the latest/desired version
```

Comment:
please use a fixed version instead, in this case it should be `1.34.0` ... we just had a problem because of the `~>` config

Reply:
I think I was able to merge all the upstream changes and keep the versions you mention here (Addons 1.34.0 and karpenter 1.0.6).

Reply:
awesome, this LGTM!